src/views/GameSettingsWizard/__tests__/GameSettingsWizard.test.tsx

```
@@ -0,0 +1,393 @@
+import { render, screen, waitFor, act } from '@/test-utils';
+import { vi, describe, it, expect, beforeEach } from 'vitest';
+import GameSettingsWizard from '../index';
+
+// Mock the child components
+vi.mock('../RoomStep', () => ({
+  default: ({ nextStep }: { nextStep: () => void }) => (
+    <div data-testid="room-step">
+      <button onClick={nextStep}>Next</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)